MDANSE/Src/MDANSE/Framework/Configurators/IConfigurator.py

```diff
@@ -124,6 +125,7 @@ def __init__(
 
         """
         self.name = name
+        self.prediction_label = kwargs.get("prediction_label", name)
```

Comment:
Prefer full args over `**args`

```
        show_prediction: bool = False,
        prediction_label: str = "",
```

```suggestion
        self.prediction_label = prediction_label or name
```